cypress/integration/options/legend.cy.js

```diff
@@ -464,7 +465,7 @@ describe('Options - Legend', () => {
 
         cy.log('navigates to the start page and adds data items')
         goToStartPage()
-        changeVisType(visTypeDisplayNames[VIS_TYPE_PIVOT_TABLE])
+        changeVisType(getDisplayNameByVisType(VIS_TYPE_PIVOT_TABLE))
```

Comment:
I guess this is not needed as it should be already Pivot table by default, same for the other ones below.
Not a problem leaving this step anyway.

Reply:
Yep, then we avoid having the same problem again later.